x-pack/elastic-agent/pkg/composable/providers/kubernetes/config.go

```diff
@@ -25,6 +27,16 @@ type Config struct {
 
 	// Needed when resource is a Pod or Node
 	Node string `config:"node"`
+
+	AddResourceMetadata *metadata.AddResourceMetadataConfig `config:"add_resource_metadata"`
+	IncludeLabels       []string                            `config:"include_labels"`
+	ExcludeLabels       []string                            `config:"exclude_labels"`
+
+	LabelsDedot      bool `config:"labels.dedot"`
+	AnnotationsDedot bool `config:"annotations.dedot"`
```

Comment:
Can this just be the default? Do we need this configurable now?

Reply:
I'm not sure about the history of these settings. Maybe some users had requests for these? @jsoriano @exekias do you think we could remove those settings and have dedoting as an always-on feature?